package.json

```diff
@@ -436,6 +436,11 @@
                     "when": "view == dockerImages && viewItem == image",
                     "group": "images_3_destructive@1"
                 },
+                {
+                    "command": "vscode-docker.images.group.remove",
+                    "when": "view == dockerImages && viewItem =~ /imageGroup$/i",
```

Comment:
Thinking about it, if the user sees both "Image Group: Remove Image..." and "Image: Remove Image..." in the command palette, confusion is likely to result. I'd suggest a couple things--

1. Don't show this command in the command palette--i.e., in the `menus.commandPalette` part of this file, add one with `"when": "never"` (there are many other examples)
2. Use the same `"category"` property as the other image commands (i.e. `"%vscode-docker.commands.category.dockerImages%"`)
3. Remove the newly-added category string from `package.nls.json`

We did a similar thing when doing commands for compose down--which is a different command if you run it on a file vs. run it on a tree item.